tests/integration/test_ourlogs.py

```diff
@@ -77,7 +77,6 @@ def test_ourlog_extraction(
         "project_id": 42,
         "retention_days": 90,
         "timestamp_nanos": int(start.timestamp() * 1e9),
-        "observed_timestamp_nanos": int(end.timestamp() * 1e9),
```

Comment:
We have helpers for this, so you can assert timestamps in tests, e.g. `time_within_delta()` should work here.

These assert/time helpers have a custom eq logic implemented which allows fuzzy-ish time assertions.

Reply:
Realized we don't have support for nanos, added it in #4562